src/app/applications/multicast/multicast-edit/multicast-edit.component.ts

```diff
@@ -181,8 +201,7 @@ export class MulticastEditComponent implements OnInit {
     this.scrollToTopService.scrollToTop();
   }
   ngOnDestroy(): void {
-    if (this.multicastSubscription) {
-      this.multicastSubscription.unsubscribe();
-    }
+    this.multicastSubscription?.unsubscribe();
   }
+  private _onDestroy = new Subject<void>();
```

Comment:
Don't mix the order of methods and fields. Class fields are expected to be at the top before any method declaration